lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -681,6 +774,77 @@ def test_no_matches_exception(self):
                 self.partial_historic_forecasts, partial_truth)
 
 
+class Test_mask_cube(SetupCubes):
+    """Test the mask_cube method"""
+
+    @ManageWarnings(
+        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def setUp(self):
+        """Set up addtional cube for land sea mask."""
+        super().setUp()
+        mask_data = np.array([[0, 1, 0],
+                              [0, 1, 1],
+                              [1, 1, 0]],
+                             dtype=np.int32)
+        self.mask_cube = set_up_variable_cube(
+            mask_data, name="land_binary_mask", units="1")
+        self.plugin = Plugin("guassian", "20171110T0000Z")
```

Comment:
Typo: `guassian` => 'gaussian'

However, this typo has pointed me to a slightly bigger issue related to #950, where we've removed the check on whether the distribution passed is valid. See the lines below:

```python
        valid_distributions = (ContinuousRankedProbabilityScoreMinimisers().	
                               minimisation_dict.keys())	
        if distribution not in valid_distributions:	
            msg = ("Given distribution {} not available. Available "	
                   "distributions are {}".format(	
                       distribution, valid_distributions))	
            raise ValueError(msg)
```
This should be fixed, probably in #950, rather than this PR, by moving this check to the `EstimateCoefficientsForEnsembleCalibration` plugin.

Reply:
Fixing it in #950 is fine by me